SPECS/grub2/grub2.spec

```diff
@@ -107,6 +107,8 @@ Patch:          sbat-4-0006-fs-ntfs-Make-code-more-readable.patch
 # The Azure Linux team created this patch since the gcc version in use at the
 # time optimizes the code incorrectly, leading to network traffic getting
 # dropped in scenarios like PXE booting.
```

Comment:
Can we move this comment to above the relevant patch? disable-checksum-code-optimization.patch?

Reply:
+1, good catch, @Camelron.